docs/resources/oracle_dsource.md

```diff
@@ -0,0 +1,197 @@
+# Resource: <resource name> delphix_oracle_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Continuous Data Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+The dSource resource allows Terraform to create and delete Delphix dSources. This specifically enables the apply and destroy Terraform commands. Modification of existing dSource resources via the apply command is not supported. All supported parameters are listed below.
```

Comment:
Old: The dSource resource allows Terraform to create and delete Delphix dSources.
New: The dSource resource allows Terraform to create and delete dSources.

Reply:
Similar update to the PostgreSQL docs